core/templates/pages/thanks-page/thanks-page.component.html

```diff
@@ -1,55 +1,63 @@
-<background-banner></background-banner>
-<div class="oppia-static-header">
-  <h1>Thank You!</h1>
-</div>
-<div class="oppia-static-content-below-banner">
-  <div class="oppia-page-card oppia-static-content protractor-test-thanks-page">
-    <div class="oppia-static-card-content oppia-static-card-content-wide oppia-default-height">
-      <h2>Thank you for donating to The Oppia Foundation!</h2>
-      <br>
-      <div class="oppia-float-left">
-        <picture>
-          <source type="image/webp" [srcset]="getStaticImageUrl('/general/donate.webp')">
-          <source type="image/png" [srcset]="getStaticImageUrl('/general/donate.png')">
-          <img [src]="getStaticImageUrl('/general/donate.png')" alt="Otter holding a gift" style="height: 250px;">
-        </picture>
-      </div>
-      <div class="oppia-float-right">
-        <p>
-          Your contribution will be used towards:
-        </p>
-        <ul style="margin-left: 30px;">
-          <li>Maintaining our servers</li>
-          <li>Marketing</li>
-          <li>Distributing learning materials to students</li>
-        </ul>
-        <p>
-          Thank you for your support!
-        </p>
+<div class="thanks-page">
```

Comment:
Any specific reason for this div? Can we use `thanks-page` tag?

Reply:
We can't do that because the use of tags to apply styles is not possible in angular 8, therefore we have to keep the code under a div to use the angular 1 type tag.